app/models/match.rb

```diff
@@ -38,15 +41,31 @@ def self.get_info team
             roster.kills = summ_info["stats"]["kills"] + roster.kills
             roster.deaths = summ_info["stats"]["deaths"] + roster.deaths
             roster.assists = summ_info["stats"]["assists"] + roster.assists
+            roster.save
             sum_damage_dealt = summ_info["stats"]["totalDamageDealtToChampions"]
+            sum_physical_dealt = summ_info["stats"]["physicalDamageDealtToChampions"]
+            sum_magic_dealt = summ_info["stats"]["magicDamageDealtToChampions"]
+            sum_true_dealt = summ_info["stats"]["trueDamageDealtToChampions"]
             team_damage_dealt = 0
+            team_physical_dealt = 0
+            team_magic_dealt = 0
```

Comment:
Useless assignment to variable - `team_magic_dealt`. Did you mean `team_damage_dealt`?